src/app/components/global-search/global-search.component.scss

```diff
@@ -0,0 +1,3 @@
+::ng-deep .modal-backdrop {
+  --bs-backdrop-bg: transparent;
```

Comment:
you don't need this. Just check the documentation of Angular material dialog

Reply:
Temporary addition. Subject to change on receiving actual design